src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -1317,6 +1338,35 @@ struct jit_uni_interpolate_kernel_f32 : public jit_uni_interpolate_kernel, publi
         }
     }
 
+    /**
+     * @brief Emulates vgatherdpd/vgatherdps instruction for byte-sized data with 32-bit indices.
+     * Mask is considered with all bits set.
+     *
+     * @param dest_addr Register with the start destination address
+     * @param source_addr Register with the start source address
+     * @param ind_addr Register with the start index address
+     * @param is_scalar Flag to perform scalar operation instead of vector
+     * @return none.
+     */
+    inline void gather_i8_i32_indices_store(const Xbyak::Reg64 &dest_addr, const Xbyak::Reg64 &source_addr,
+                                            const Xbyak::Reg64 &ind_addr, bool is_scalar) {
+        constexpr int gpr_size = 8;
+        sub(rsp, gpr_size);
+        // move content in register to content in address(ptr[])
+        mov(ptr[rsp], reg_tmp_64);
```

Comment:
Let remove these lines, I believe it's safe in current code path.

Reply:
Done